packages/loot-core/package.json

```diff
@@ -58,6 +59,7 @@
     "fake-indexeddb": "^3.1.8",
     "fast-check": "3.15.1",
     "i18next": "^23.11.5",
+    "jest": "^29.7.0",
```

Comment:
Since we switched to vitest, it seems a little weird to keep jest as a dev dependency still. Is there any other snapshot library that's not jest-specific?

Reply:
There doesn't look to be anything completely drop-in that I can find. We could probably roll our own or tweak another though.

Reply:
Makes sense. Fine to just make a ticket for later.

Reply:
https://github.com/actualbudget/actual/pull/5024

Reply:
https://github.com/actualbudget/actual/issues/5026